src/ubuntu/24.04/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt-get update \
         clang-14 \
         cmake \
         gdb \
+        iproute2 \
```

Comment:
This probably belongs in the `# Install tools used by the VSO build automation.` section below.